src/utils/notifications.ts

```diff
@@ -72,6 +72,23 @@ export const handlePushManagerUnsubscribe = async (hide: () => void) => {
   }
 };
 
+export const blockNotification = async () => {
+  const data = {
+    accountId: 'erditkurteshi.near',
+    endpoint: '/preferences/set',
+    dapp: 'mention',
+    block: true,
+  };
```

Comment:
@erditkurteshiSQA could you please explain what this fields means please? I far as I understood:

- `accountId` - means which account you want to block notifications from. If this is right then this field should be passed as property;
- `endpoint` - (don't need to change here);
- `dapp` - is this means the type of notification? For example "like", "comment", ... etc? If so then this field should be passed as a property too;
- `block` - no change here. Or is there a similar endpoint where we can unblock this?

Reply:
You're correct, now I have passed the needed data as a property where applicable.